packages/nocodb/src/lib/models/Model.ts

```diff
@@ -399,13 +399,18 @@ export default class Model implements TableType {
     return true;
   }
 
-  async mapAliasToColumn(data, sanitize = v => v) {
+  async mapAliasToColumn(data) {
     const insertObj = {};
     for (const col of await this.getColumns()) {
       if (isVirtualCol(col)) continue;
-      const val =
-        data?.[sanitize(col.column_name)] ?? data?.[sanitize(col.title)];
-      if (val !== undefined) insertObj[sanitize(col.column_name)] = val;
+      const val = data?.[col.column_name] ?? data?.[col.title];
+      if (val !== undefined) {
+        insertObj[
+          col.column_name.replace(/([^\\]|^)(\?+)/g, (_, m1, m2) => {
```

Comment:
You can use the `sanitize` method here instead of repeating

Reply:
Good spot. Forget to update this one. Just updated.